lib/xcpretty/parser.rb

```diff
@@ -568,7 +568,7 @@ def update_linker_failure_state(text)
       when LINKER_UNDEFINED_SYMBOL_LOCATION_MATCHER
         current_linker_failure[:body] << $1.rstrip
       when LINKER_DUPLICATE_SYMBOLS_LOCATION_MATCHER
-        current_linker_failure[:body] << "    " + $1.rstrip.split("/").last
```

Comment:
keep it, fix the tests

Reply:
maybe keep the rstrip if you want

Reply:
I thought that we want to show the full path of the file, not just the filename. The `rstrip`'s still there.

Reply:
Let's do just the file.

Reply:
Reverted.